cashctrl_ledger/ledger.py

```diff
@@ -85,11 +85,12 @@ def dump_to_zip(self, archive_path: str):
             archive.writestr('accounts.csv', self.accounts.list().to_csv(index=False))
             archive.writestr('price_history.csv', self.price_history.list().to_csv(index=False))
             archive.writestr('ledger.csv', self.ledger.list().to_csv(index=False))
-            archive.writestr('assets.csv', self.price_history.list().to_csv(index=False))
+            archive.writestr('assets.csv', self.assets.list().to_csv(index=False))
 
     def restore_from_zip(self, archive_path: str):
```

Comment:
Can you refresh my mind why we need a `restore_from_zip` method in CashCtrlLedger? 
- Is it because `revaluations` can not be restored?
- Can we not just drop this method? `restore` will then raise an error if revaluations are present.

Reply:
It's because then parent `restore_from_zip()` method will look into unexistent `revaluations.csv` file with `revaluations = pd.read_csv(archive.open('revaluations.csv'))` [line](https://github.com/macxred/pyledger/blob/62a9f79d2a09f02a8fb8ca07e0fd3ad81d1ff534/pyledger/ledger_engine.py#L199).

Reply:
Ok, we can leave it this way.

I guess the alternative would be to change the `restore_from_zip` method to pass `None` to `restore` if a given file is not present.